### ISSUE TITLE
dev-util/amdapp-3.0_beta can't be fetched anymore

### ISSUE THREAD

**Issue by EoD** [gentoo-science/sci#542]
The amdapp ebuild is updated and the fetch-restricted files cannot be obtained anymore.

I tried using the newly released files for AMD-APP 3.0, but they seem not to be compatible with the current ebuild. They also got renamed:
```
-rw-r----- 1 portage portage 175232929 Nov 27 11:01 /usr/portage/distfiles/AMD-APP-SDKInstaller-v3.0.130.135-GA-linux32.tar.bz2
-rw-r----- 1 portage portage 175434609 Nov 27 11:01 /usr/portage/distfiles/AMD-APP-SDKInstaller-v3.0.130.135-GA-linux64.tar.bz2
```

I tried renaming the files at the beginning of the ebuild
```
MY_V="$(get_version_component_range 1).$(get_version_component_range 2).130.135-GA"

X86_AT="AMD-APP-SDKInstaller-v${MY_V}-linux32.tar.bz2"
AMD64_AT="AMD-APP-SDKInstaller-v${MY_V}-linux64.tar.bz2"

MY_P_AMD64="AMD-APP-SDKInstaller-v${MY_V}-linux64.sh"
MY_P_AMD32="AMD-APP-SDKInstaller-v${MY_V}-linux32.sh"
```

But this seems not to be sufficient to update the ebuild to 3.0 release
```log
>>> Unpacking source...
>>> Unpacking AMD-APP-SDKInstaller-v3.0.130.135-GA-linux64.tar.bz2 to /var/tmp/portage/dev-util/amdapp-3.0_beta-r1/work
head: cannot open ‘’ for reading: No such file or directory
>>> Unpacking ./AMD-APP-SDKInstaller-v3.0.130.135-GA-linux64.sh to /var/tmp/portage/dev-util/amdapp-3.0_beta-r1/work
 * ERROR: dev-util/amdapp-3.0_beta-r1::science failed (unpack phase):
 *   unpack: ./AMD-APP-SDKInstaller-v3.0.130.135-GA-linux64.sh does not exist
```

**Comment by marbre** [comment]
Fixed with commit e64d46f. You were close the the solution, but it was not necessary to adjust `MY_P_AMD64` and `MY_P_AMD32`.

**Comment by EoD** [comment]
Oh. Thanks!
